oadm panic with no args

<!--
Note that this is the issue tracker for OpenShift Origin.

OpenShift Installer issues should be reported at https://github.com/openshift/openshift-ansible/issues
Management Console issues are collected at https://github.com/openshift/origin-web-console/issues
Documentation issues are better reported at https://github.com/openshift/openshift-docs/issues
-->

[provide a description of the issue]
$oadm
DEPRECATED: The 'oadm' command is deprecated, please use 'oc adm' instead.
panic: runtime error: index out of range

goroutine 1 [running]:
main.main()
	/builddir/build/BUILD/atomic-openshift-git-0.8f8b69d/_output/local/go/src/github.com/openshift/origin/cmd/oadm/main.go:31 +0x471

##### Version
[provide output of the `openshift version` or `oc version` command]
oc v3.9.0-0.36.0
kubernetes v1.9.1+a0ce1bc657
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://aos-143.lab.sjc.redhat.com:8443
openshift v3.9.0-0.36.0
kubernetes v1.9.1+a0ce1bc657

##### Steps To Reproduce
1. [step 1]
2. [step 2]

##### Current Result

##### Expected Result

##### Additional Information
[try to run `$ oc adm diagnostics` (or `oadm diagnostics`) command if possible]
[if you are reporting issue related to builds, provide build logs with `BUILD_LOGLEVEL=5`]
[consider attaching output of the `$ oc get all -o json -n <namespace>` command to the issue]
[visit https://docs.openshift.org/latest/welcome/index.html]

--- a/cmd/oadm/main.go
+++ b/cmd/oadm/main.go
@@ -26,9 +26,9 @@ func main() {
 
 	admCmd := ""
 
-	// there is no `oc adm version` command.
-	// special-case it here.
-	if os.Args[1] != "version" {
+	// 1. there is no `oc adm version` command,special-case it here.
+	// 2. oadm with no args should run oc adm and print out the default usage msg.
+	if (len(os.Args) > 1 && os.Args[1] != "version") || len(os.Args) == 1 {
 		admCmd = "adm"
 	}
 

